src/BeamAdapter/component/forcefield/AdaptiveBeamForceFieldAndMass.inl

```diff
@@ -362,6 +364,30 @@ void AdaptiveBeamForceFieldAndMass<DataTypes>::addMToMatrix(const MechanicalPara
     }
 }
 
+template<class DataTypes>
+void AdaptiveBeamForceFieldAndMass<DataTypes>::buildMassMatrix(sofa::core::behavior::MassMatrixAccumulator* matrices)
+{
+    unsigned int numBeams = l_interpolation->getNumBeams();
+
+    for (unsigned int b=0; b<numBeams; b++)
+    {
+        unsigned int node0Idx, node1Idx;
+        BeamLocalMatrices &bLM = m_localBeamMatrices[b];
+        l_interpolation->getNodeIndices( b,  node0Idx, node1Idx );
+
+        /// matrices in global frame
+        const Matrix6x6 M00 = bLM.m_A0Ref.multTranspose((bLM.m_M00 * bLM.m_A0Ref));
+        const Matrix6x6 M01 = bLM.m_A0Ref.multTranspose((bLM.m_M01 * bLM.m_A1Ref));
+        const Matrix6x6 M10 = bLM.m_A1Ref.multTranspose((bLM.m_M10 * bLM.m_A0Ref));
+        const Matrix6x6 M11 = bLM.m_A1Ref.multTranspose((bLM.m_M11 * bLM.m_A1Ref));
+
+        matrices->add(node0Idx * 6, node0Idx * 6,  M00);

```

Comment:
There is no Mat6x6 version of add in https://github.com/sofa-framework/sofa/blob/master/Sofa/framework/Core/src/sofa/core/MatrixAccumulator.h
Maybe did you add it in your sofa local branch and would need to PR it in https://github.com/sofa-framework/sofa ? @EulalieCoevoet

Reply:
Indeed !